ocaml/parsing/builtin_attributes.mli

```diff
@@ -162,12 +162,11 @@ val parse_standard_interface_attributes : Parsetree.attribute -> unit
 val parse_standard_implementation_attributes : Parsetree.attribute -> unit
 
 val has_local_opt: Parsetree.attributes -> bool
-val has_curry: Parsetree.attributes -> bool
+
+val is_explicitly_curried : Parsetree.attributes -> bool
```

Comment:
What's the difference between `is_explicitly_curried` and `has_curry`? (Or `has_local_opt`?) This question may well be answered with a comment.

Reply:
No intentional difference, stuff just got deleted/reappeared and I wasn't careful with names.  This is for handling `[@curry]` on terms, though, which we're going to delete anyway.